README.md

```diff
@@ -1,6 +1,11 @@
-[![Build Status](https://github.com/SpiNNakerManchester/DataSpecification/workflows/Python%20Actions/badge.svg?branch=master)](https://github.com/SpiNNakerManchester/DataSpecification/actions?query=workflow%3A%22Python+Actions%22+branch%3Amaster)
-[![Documentation Status](https://readthedocs.org/projects/dataspecification/badge/?version=latest)](https://dataspecification.readthedocs.io/en/latest/?badge=latest)
-[![Coverage Status](https://coveralls.io/repos/github/SpiNNakerManchester/DataSpecification/badge.svg?branch=master)](https://coveralls.io/github/SpiNNakerManchester/DataSpecification?branch=master)
+<div align="center">
+  
+  ![](https://vectr.com/kerff/ddbmvyZmm.svg?width=600&height=300&select=aNbKxciPh)
```

Comment:
This image appears to be broken. Please substitute with something that actually works.